packages/cli/src/commands/start.ts

```diff
@@ -714,22 +714,20 @@ const startAgents = async (options: {
         }
 
         const startedAgents = [];
-        for (const agent of agents) {
-          try {
+        const results = await Promise.allSettled(
```

Comment:
will parallel loading of the character make the logs harder for devs to read as they'll have to piece together the entries.

Reply:
Yes but starting up is > 1 minute right now and its incredibly frustrating to dev on